src/main/java/com/cinema/controllers/MovieController.java

```diff
@@ -0,0 +1,39 @@
+package com.cinema.controllers;
+
+import com.cinema.model.Movie;
+import com.cinema.model.dto.MovieDto;
+import com.cinema.service.MovieService;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/movies")
+public class MovieController {
+    private final MovieService movieService;
+    private final ModelMapper modelMapper;
+
+    @Autowired
+    public MovieController(MovieService movieService, ModelMapper modelMapper) {
+        this.movieService = movieService;
+        this.modelMapper = modelMapper;
+    }
+
+    @GetMapping("/all")
```

Comment:
I think "/all"  is unnecessary (REST Resource Naming Guide: Never use CRUD function names in URIs)

Reply:
done